lang/gcc-devel/Portfile

```diff
@@ -6,9 +6,16 @@ PortGroup           compiler_blacklist_versions  1.0
 PortGroup           active_variants              1.1
 PortGroup           conflicts_build              1.0
 
-epoch               4
+# for devel
+PortGroup github    1.0
+github.setup        iains gcc-darwin-arm64 6b589e6cb14fb74eaac99411a64083ff32fada85
+
+checksums           rmd160  005cdb1c71ee78234d3db08853e873cbae320854 \
+                    sha256  fe70b2caedf6f23b1caa9bf478b195be7d6801bd10836fb2419c85446034bbe9 \
+                    size    121028082
+
```

Comment:
still have no idea why this didn't work for Michael when he tried it -- must have used the wrong commit hash I guess, as there is no other reason...